crates/core/src/db/queries/listings_triple_join.rs

```diff
@@ -58,10 +63,13 @@ pub fn rejected(now: NaiveDateTime) -> Rejected {
         .is_not_null()
         .and(listings::price_floor.gt(too_expensive));
 
-    listings::ended.eq(false).and(not(already_ended
-        .or(ended_instant_sale)
-        .or(not_ending_soon)
-        .or(too_expensive)))
+    listings::ended
+        .eq(false)
+        .and(not(already_ended
+            .or(ended_instant_sale)
+            .or(not_ending_soon)
+            .or(too_expensive)))
```

Comment:
love how this reads.

Reply:
Yeah, breaking that up into variables ended up making the query scan cleaner in addition to making writing the resulting DSL type easier.